libnmstate/nm/sriov.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+from libnmstate.error import NmstateNotImplementedError
+from libnmstate.schema import Ethernet
+from libnmstate.schema import SRIOV
+
+
+def create_setting(iface_state):
+    setting = None
+    ethernet_setting = iface_state.get(Ethernet.CONFIG_SUBTREE)
+    if ethernet_setting:
```

Comment:
You can simplify this as

```
setting = None
sriov_config = iface_state.get(Ethernet.CONFIG_SUBTREE, {}).get(SRIOV.CONFIG_SUBTREE)
if sriov_config:
    raise NmstateNotImplementedError

return setting
```
This avoids the multiple ifs. Hope this will please the linter.

Reply:
Yes, that is fine. Thanks!